weasyl/login.py

```diff
@@ -72,7 +72,7 @@ def authenticate_bcrypt(username, password, session=True):
 
     d.metric('increment', 'attemptedlogins')
 
-    unicode_success = bcrypt.checkpw(password.encode('utf-8'), HASHSUM)
+    unicode_success = bcrypt.hashpw(password.encode("utf-8"), HASHSUM.encode("utf-8"))
```

Comment:
While letting everybody log in with any password arguably makes for a nice user experience, I think we should focus more on security here.

`hashpw` returns a hash digest; you need to compare it against the existing one.